homeassistant/components/influxdb/sensor.py

```diff
@@ -123,6 +189,14 @@ def validate_query_format_for_version(conf: dict) -> dict:
             vol.Required(CONF_WHERE): cv.template,
         }
     ),
+    LANGUAGE_RAW: _QUERY_SENSOR_SCHEMA.extend(
+        {
+            vol.Optional(CONF_DB_NAME): cv.string,
+            vol.Optional(CONF_BUCKET): cv.string,
+            vol.Required(CONF_QUERY): cv.template,
+            vol.Optional(CONF_FIELD, default=INFLUX_CONF_VALUE_V2): cv.string,
```

Comment:
```suggestion
            vol.Optional(CONF_FIELD, default=DEFAULT_FIELD): cv.string,
```
I would not default to the V2 field here. I would default this to the V1 field (which is `"value"` looks like). For V2 just completely ignore this config option and always use `_value`. We'll need to make sure the doc notes that this field is only for V1.

Reply:
OK. As the `CONF_FIELD` won't be valid for V2, the default value for V1 will be set up in the `validate_raw_version_config` function.